tck/README.md

```diff
@@ -243,11 +243,17 @@ Subscriber rules Verification is split up into two files (styles) of tests.
 
 The Blackbox Verification tests do not require the implementation under test to be modified at all, yet they are *not* able to verify most rules. In Whitebox Verification, more control over `request()` calls etc. is required in order to validate rules more precisely.
 
+It is highly recommended to implement the `SubscriberBlackboxVerification<T>` instead of the blackbox one even if it is more troublesome to do so, as it can test far more corner cases in implementations that would otherwise be left untested (if only blackbox tests would be used).
+
 ### createElement and Helper Publisher implementations
 Since testing a `Subscriber` is not possible without a corresponding `Publisher` the TCK Subscriber Verifications
 both provide a default "*helper publisher*" to drive its test and also alow to replace this Publisher with a custom implementation.
 The helper publisher is an asynchronous publisher by default - meaning that your subscriber can not blindly assume single threaded execution.
 
+When extending subscriber verification classes a type parameter representing the element type passed through the stream must be given. MostImplementations usually should not be sensitive to the type of element being signalled, but sometimes a Subscriber may be limited to only be able to work within a known set of types - like a `FileSubscriber extends Subscriber<String>` for example, that writes each element it receives as new line into a file. 
+For element type agnostic Subscribers it is the simplest to parameterize the tests using `Integer` and in the `createElement(int idx)` method (explained below in futher detail), return the incoming `int`. 
+In case an implementation needs to work on a specific type, the verification class should be parameterized using that type (e.g. `class StringSubTest extends SubscriberBlackboxVerification<String>`) and a `String` must be returned from the `createElement` method.
```

Comment:
Perhaps avoid using blackbox verification in the examples here if we want to encourage the use of whitebox?